pkg/helpers/contexts/heartbeat.go

```diff
@@ -36,10 +36,10 @@ func WithHeartBeat(parent context.Context, timeout time.Duration, lg log.Entry)
 		HeartBeat:     ctx.heartBeat,
 		HeartBeatStop: ctx.heartBeatStop,
 	}
-	ctx.Context, actuator.Cancel = context.WithCancel(parent)
+	ctx.Context, ctx.cancel = context.WithCancel(parent)
```

Comment:
`actuator.Cancel()` wasn't used outside of this struct, right?

Reply:
Only in tests which it has been replaced by `HeartBeatStop`. I think that if used, compilation tests would have failed.